source/trackers.json

```diff
@@ -1200,6 +1212,7 @@
         "iq.com": "iqiyi",
         "ironsrc.com": "ironsource",
         "ironsrc.net": "ironsource",
+        "soom.la": "ironsource",
```

Comment:
Is the domain `soom.la` from `ironsource` tracker or from `soomla` ?

Reply:
![image](https://github.com/AdguardTeam/companiesdb/assets/8361299/d4f0246d-a38b-4d76-b56e-219ddee65c37)

Reply:
not sure if we need their old domains, which redirect to is.com

Reply:
In general this is a regular site. We can't add all existing sites to our base.